pkgs/development/tools/misc/binutils/default.nix

```diff
@@ -48,11 +49,6 @@ stdenv.mkDerivation {
     # Make binutils output deterministic by default.
     ./deterministic.patch
 
-    # Help bfd choose between elf32-littlearm, elf32-littlearm-symbian, and
-    # elf32-littlearm-vxworks in favor of the first.
-    # https://github.com/NixOS/nixpkgs/pull/30484#issuecomment-345472766
-    ./disambiguate-arm-targets.patch
-
```

Comment:
I'm not certain that this patch is no longer needed. I inspected the source and it _seems_ like it isn't, but a second opinion would be good.